lib/ansible/plugins/cliconf/enos.py

```diff
@@ -1,34 +1,23 @@
-# This code is part of Ansible, but is an independent component.
-# This particular file snippet, and this file snippet only, is BSD licensed.
-# Modules you write using this snippet, which is embedded dynamically by
-# Ansible still belong to the author of the module, and may assign their own
-# license to the complete work.
 #
 # Copyright (C) 2017 Lenovo, Inc.
```

Comment:
Please update as below in sync with other community supported platforms.
(c) 2017 Red Hat Inc.
#
(c) 2017 Lenovo, Inc.